PcdcAnalysisTools/blueprint/routes/views/counts/__init__.py

```diff
@@ -5,20 +5,31 @@
 from PcdcAnalysisTools import utils
 from PcdcAnalysisTools import auth
 
+from pcdcutils.environment import is_env_enabled
+import json
+import os 
+
+from dotenv import load_dotenv
+
+load_dotenv()
```

Comment:
The ENV variable will be set by Kubernetes, so the load_dotenv is not necessary here.